import datetime

try:
    try:
        print(a)
    except Exception as e:
        print(e)
    print(b)
except Exception as e:
    print(e)